### ISSUE TITLE
off-by-one in the docs for LineColumn

### ISSUE THREAD

**Issue by samuelpilz** [dtolnay/proc-macro2#326]
This crates docs for [`LineColumn`](https://docs.rs/proc-macro2/latest/proc_macro2/struct.LineColumn.html) say that the column is 0-indexed. The docs for [`proc_macro::LineColumn`](https://doc.rust-lang.org/proc_macro/struct.LineColumn.html) say that the column is 1-indexed. As far as I can see, this is a documentation issue in proc_macro2 and the column value from the `proc_macro::LineColumn` is used without offset edit.

**Comment by samuelpilz** [comment]
Looking at the tests, when using the `parse::<TokenStream>` method outside a proc-macro environment, the column seems to be 0-indexed. PR #372 shows that they are 1-indexed in a proc-macro environment.

**Comment by dtolnay** [comment]
Yeah it seems this got changed in libproc_macro in https://github.com/rust-lang/rust/pull/87712.

**Comment by samuelpilz** [comment]
can I help fixing this?
